ideas/58-mainnet.md

```diff
@@ -0,0 +1,72 @@
+## Preamble
+
+    Idea: 58-mainnet-default
+    Title: Use Mainnet as the default network
+    Status: In progress
+    Created: 2017-12-08
+
+
+## Summary
+Alpha builds of Status should default to Mainnet. We should be using Status in a real-world environment.
+
+## Swarm Participants
+
+- Lead Contributor: @adambabik
+- Testing & Evaluation: <!-- @username -->
+- Contributor: @themue
+- Contributor: <!-- @username -->
+- PM: @chadyj
+
+## Product Overview
+
+Currently the Status alpha builds use testnet, but using Mainnet is a key milestone for the public launch of Status. There will be potentially major performance and security implications so the goal is to surface these issues early enough by testing the app in a real-world environment.
+
+From a users perspective, Mainnet should be a default network as it is required by most popular DApps, and is needed for real transactions, so the full potential of Status can't be seen until we switch. The other networks will continue to be available for developers and testers.
```

Comment:
Would rephrase last sentence: _"The other networks will continue to be available, ie for developers and tests."_. It sounds less restrictive.